src/Kunstmaan/NodeBundle/Repository/NodeTranslationRepository.php

```diff
@@ -296,7 +296,7 @@ public function createNodeTranslationFor(HasNodeInterface $hasNode, $lang, Node
             ->setNode($node)
             ->setLang($lang)
             ->setTitle($hasNode->getTitle())
-            ->setSlug(Slugifier::slugify($hasNode->getTitle(), ''))
+            ->setSlug($hasNode->getTitle(), '')
```

Comment:
I've removed the slugify call there because I was not able to acces the service in that class, but there is a listener that listens for the doctrine.event_listener event that gets dispatched. In that listener (NodeTranslationListener) I set the slug to the correct slugified slug.